codegenerator/cli/templates/static/codegen/src/eventFetching/ChainFetcher.res

```diff
@@ -131,10 +131,9 @@ let makeFromDbState = async (chainConfig: Config.chainConfig, ~maxAddrInPartitio
       {
         FetchState.filter: qItem => {
           //Only keep events greater than the last processed event
-          (qItem.chain->ChainMap.Chain.toChainId, qItem.blockNumber, qItem.logIndex) >
-          (event.chainId, event.blockNumber, event.logIndex)
+          qItem.blockNumber > event.blockNumber || qItem.logIndex > event.logIndex
```

Comment:
Unfortuntaley this won't work multichain

Reply:
Why? We have a separate chain fetcher for each chain